src/util/denormalsarezero.h

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Because the engine thread is created from external libraries, probably ALSA, where we cannot set the CC flags.